task/stackrox-image-scan/0.1/stackrox-image-scan.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: stackrox-image-scan
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/tags: security
+    tekton.dev/displayName: "Scan an image with StackRox/RHACS"
```

Comment:
can we please add one more annotation `tekton.dev/categories` similar to other tasks in the catalog?